doc/user/load_save.md

```diff
@@ -527,7 +527,8 @@ mlpack::data::Save(outImages, matrix, info);
 
 It is possible to resize images in mlpack with the following function:
 
-- `ResizeImages(images, info, newWidth, newHeight)`
+- `ResizeImages(images, info, newWidth, newHeight, clamp = false, minVal = 0.0,
+   maxValue = 255.0)`
```

Comment:
```suggestion
- `ResizeImages(images, info, newWidth, newHeight, clamp = false, minVal = 0.0, maxValue = 255.0)`
```

These actually have to be on one line, otherwise they will not be rendered correctly in the HTML.